### ISSUE TITLE
Node 6.x breaking changes in the path module

### ISSUE THREAD

**Issue by ow--** [dman777/gulp-newy#4]
6.x seems destined to break backwards compatibility in the `path` module and enforces string checks on paths (nodejs/node#5348). This of course breaks newy when the callback returns the path to a non-existent file, as would often be the case in transpiling build chains.

Lovely little plugin though. Works exactly how you want it to.

**Comment by dman777** [comment]
Thank you so much for the heads up and the complement! I will get started on that today to make it work with 6.0.

**Comment by dman777** [comment]
Ok, this is fixed released as  `version 1.1.2` and should be good to go. Changes made were:

```javascript
 var projectDir = path.resolve(__dirname, "../../../");
```
and
```javascript
function consoleOut(absSrcFile, destinationFile) {
    console.log("-------------------- Newy Message ------------------" .magenta);
    console.log("File " .magenta + path.basename(absSrcFile) .white +
                        " is new/newer then " .magenta + path.basename(String(destinationFile))
            .white);
    console.log("----------------------------------------------------" .magenta);
}
```

Thank you again for filing this issue! :smile_cat:  Please let me know if you find anymore.

**Comment by ow--** [comment]
Great! Thank you! :smile:
